src/test/java/org/apache/commons/imaging/common/bytesource/ByteSourceTest.java

```diff
@@ -17,24 +17,22 @@
 
 package org.apache.commons.imaging.common.bytesource;
 
-import static org.junit.Assert.assertTrue;
+import org.apache.commons.imaging.ImagingTest;
+import org.junit.Test;
 
-import java.io.BufferedOutputStream;
-import java.io.ByteArrayOutputStream;
-import java.io.File;
-import java.io.FileOutputStream;
-import java.io.IOException;
-import java.io.OutputStream;
+import java.io.*;
 
-import org.apache.commons.imaging.ImagingTest;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
```

Comment:
I'd keep the ordering and style of the imports as before; no need to rearrange them just to import `assertEquals` and `fail`.